Improve "how to get in touch" section (#249)
codewars/docs: content/community/moderation.md

> They can be recognized by the symbol of a shield (🛡️) displayed next to their username.

Reviewer: Took the SVG from CW. Will see if it works, did test in the devtools...
```suggestion
They can be recognized by the symbol of a shield (<svg height="20" fill="none" style="display: inline-block" stroke="currentColor" stroke-width="2" viewBox="0 0 24 24"><path d="M12 2a12 12 0 0 0 8.5 3a12 12 0 0 1 -8.5 15a12 12 0 0 1 -8.5 -15a12 12 0 0 0 8.5 -3"></path></svg>) displayed next to their username.
```

Reviewer 2: for kazk:

`<svg class="inline-block" fill="currentColor" height="15" stroke="currentColor" stroke-width="1" viewBox="0 0 24 24" width="15"><path d="M12 4l4 6l5 -4l-2 10h-14l-2 -10l5 4z"></path></svg>`

Reviewer: Indeed, I forgot that this was made with Tailwind...

Author: I will revert it for now, because I am not really sure of some things. I wonder if absolute sizing or putting SVG inline are good ideas, if styling/theming will be handled correctly, etc.

If you have the crown SVG too, then guys create separate PR and we will see both icons in the preview.

## How to get in touch

> Many members of the moderation team (but not all of them) can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.

Reviewer: All of the community team has Gitter...

Author: Today, yes. But in general some might not have, or might be not active on Gitter.

Author: ```suggestion
Many members of the moderation team can be reached via [Gitter](https://gitter.im/Codewars/codewars.com). Message them directly or post on one of the public channels if you need their assistance.
```

> Issues related to specific content (kata, translations, or solutions) which can be addressed only by moderators and admins can be reported on [content issues board](https://github.com/codewars/content-issues/issues). It's not completely organized yet, but new issues can be posted there and they will be dispatched appropriately.

Reviewer: ```suggestion
Issues related to specific content (kata, translations, or solutions) that can be addressed only by moderators and admins can be reported on the [content issues board](https://github.com/codewars/content-issues/issues). It's not completely organized yet, but new issues can be posted there and they will be dispatched appropriately.
```
